File: python/class 5/question 4.py
hours_worked=int(input('Please enter the number of hours that you worked: '))
parts_cost=int(input('Please enter the cost of the parts: '))
if hours_worked==1 and parts_cost<50:
  print(f'$150')
elif hours_worked==1:
  print(f'${hours_worked*100+parts_cost}')
elif hours_worked>=2:
  print(f'${(hours_worked*100)+parts_cost}')
elif hours_worked == 0:
  print(f'$150')

File: java/class 5/Question4.java
import java.util.Scanner;

public class Question4 {
  public static void main(String[] args) {
    Scanner input = new Scanner(System.in);
    System.out.print("Enter the number of hours worked: ");
    double hours = input.nextDouble();
    System.out.print("Enter the cost of parts: ");
    double parts = input.nextDouble();
    double total = hours * 100 + parts;
    if (total < 150) {
        System.out.println("The total is $150");
      } else {
          System.out.println("The total cost is: " + total);
      }
    input.close();
  }
}
